app/controller/team.controller.js

```diff
@@ -0,0 +1,87 @@
+const db = require('../config/db.config.js');
+const Team = db.teams;
+
+//post a Team
```

Comment:
This can be more flexible and adaptable later if you use class syntax and methods for that uses cases, but if you want to use serverless architecture this is the form